model/task_lifecycle.go

```diff
@@ -817,6 +818,16 @@ func UpdateUnblockedDependencies(t *task.Task) error {
 		if err := UpdateUnblockedDependencies(&blockedTask); err != nil {
 			return errors.WithStack(err)
 		}
+
+		buildsToUpdate[blockedTask.BuildId] = true
+	}
+
+	var buildIds []string
+	for buildId := range buildsToUpdate {
+		buildIds = append(buildIds, buildId)
+	}
+	if err := UpdateVersionAndPatchStatusForBuilds(buildIds); err != nil {
+		return errors.Wrapf(err, "updating version and patch status for builds")
```

Comment:
This is updating all the build and higher statuses (build, version, and patch), so I would include the mention of build in the error message so it's a little more obvious.

Reply:
done